users/templates/users/profile_settings.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags %}
+{% block content %}
+<link rel='stylesheet' type='text/css' href="{% static 'css/profile-settings.css' %}"/>
```

Comment:
I would create a `style` block in the base template and add the style within that block. As we've done in the course.